src/server/cdn-controller.js

```diff
@@ -29,6 +30,18 @@ class CdnController {
       return res.send({ objects });
     });
 
+    router.delete('/api/v1/cdn/objects/:objectName', [needsPermission(permissions.DELETE_CDN_FILE), validateQuery(deleteObjectQuerySchema)], async (req, res) => {
+      const objectName = req.params.objectName;
+      const prefix = req.query.prefix;
+      if (!objectName) {
+        throw new BadRequest();
+      }
```

Comment:
There could also be a `validateParams` for this, like for the query.

Reply:
Fixed, thanks!